### DISCUSSION TITLE
How to deal with episode id when series title contain numbers ?

### DISCUSSION THREAD

**Discussion by rosystain** [Flexget/Flexget#3178] [Q&A]
For example：
```
series_parser    : <SeriesParseResult(data=86－不存在的战区－ / 86 EIGHTY SIX - 12 [WEB-DL][1080p][AVC AAC][CHT][MP4],name=86－不存在的战区,id=86,season=0,season_pack=None,episode=86,quality=1080p webdl aac,proper=0,special=False,status=OK)>
```
The right id is '12' but the series plugin consider '86' as episode id.
I can avoid this by stripping digits with manipulate simply but it might reducer the accuracy of matching.
Then I try with sequence_regexp but nothing changed.
```
- 86－不存在的战区:
     alternate_name: 86 EIGHTY SIX
     identified_by: sequence
     sequence_regexp: -.(\d+)
```

Did I miss something?

**Comment by lamski** [comment]
If the series name contains numbers, enclose the entire name in single quotes e.g. `'Forever 2018'`
